metashare/api/jobs.py

```diff
@@ -92,21 +92,22 @@ def alert_user_about_expiring_org(*, org, days):
     # and has unsaved changes
     get_unsaved_changes(org)
     if org.unsaved_changes:
-        # just to ensure it's properly formatted
-        raw_domain = Site.objects.first().domain
-        domain = furl(raw_domain).netloc or raw_domain
+        task = org.task
+        project = task.project
+        repo = project.repository
 
+        domain = Site.objects.first().domain
         should_be_http = not settings.SECURE_SSL_REDIRECT or domain.startswith(
             "localhost"
         )
         scheme = "http" if should_be_http else "https"
+        # just to ensure it's properly formatted
+        metashare_link = (
+            furl(f"{scheme}://{domain}")
```

Comment:
I would expect this to fail in the use-case you were concerned about, where someone has put a full URL in for the domain. If I run `furl("https://https://example.com/") I get a `ValueError`. That's why I had written it the way I had initially; to handle a bare domain or a URI. Does this work for you?

Reply:
That's true. But before, e.g. `furl('example.com').netloc` always returns empty, so wasn't doing anything. I'm happy for you to propose something that gracefully handles all the cases.

Reply:
Ah, that's intentional. Here's the flow:
 - If `furl().netloc` gets a full URI (schema and netloc at least), it will pull out the netloc, and return that.
 - if `furl().netloc` gets a raw domain, it will return `None` and then the other half of the `or` will activate, giving you the raw domain back.

Reply:
I see -- but that breaks with a trailing slash, e.g. `example.com/`.

Reply:
It had not occurred to me that someone might put in no scheme but a trailing slash 🤦‍♂ 

I'm not sure how far we should go down the defensive coding route before just trying to insist that people put correctly formatted data in.

Reply:
Sure -- I think we should either code it defensively for these two or three edge cases, or add an actual validator to the admin.

Reply:
@jgerigmeyer OK, I figured out how to monkeypatch the validation in to the Sites app, and I'm making a PR with that. I'm leaving this code as-is because it still does valuable work with the use of `furl().path` etc, but now we can trust that any newly-entered domain in the admin will be a bare domain, no path component or scheme component.